web/themes/joinup/prototype/sass/components/_tab.sass

```diff
@@ -7,101 +7,97 @@
 // Component
 // --------------------------------------------------
 .tab
-  background-color: $color-grey-light
-
-.tabs
-  border-bottom: 1px solid $color-grey-light
+  background-color: transparent
```

Comment:
Agree, but let's wait for default drupal tabs movement and then we'll adjust the spacing.